Cargo.toml

```diff
@@ -33,9 +33,9 @@ nanorand = { version = "0.7", features = ["getrandom"], optional = true }
 #flume-test = { path = "../flume-test" }
 crossbeam-channel = "0.5.5"
 crossbeam-utils = "0.8.10"
-criterion = "0.3.4"
+criterion = { version = "0.5.1", default-features = false }
```

Comment:
default features needs to be disabled to avoid
```
   Compiling criterion v0.5.1
error: Rayon cannot be used when targeting wasi32. Try disabling default features.
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/criterion-0.5.1/src/lib.rs:31:1
   |
31 | compile_error!("Rayon cannot be used when targeting wasi32. Try disabling default features.");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: could not compile `criterion` (lib) due to 1 previous error
```

Reply:
@zesterer , the workflows should be green now.